vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/overview/components/ProjectCollection.java

```diff
@@ -0,0 +1,191 @@
+import com.vaadin.flow.component.grid.Grid;
+import com.vaadin.flow.component.grid.dataview.GridLazyDataView;
+import com.vaadin.flow.component.html.Anchor;
+import com.vaadin.flow.component.html.Div;
+import com.vaadin.flow.component.icon.VaadinIcon;
+import com.vaadin.flow.component.textfield.TextField;
+import com.vaadin.flow.data.provider.SortDirection;
+import com.vaadin.flow.data.renderer.ComponentRenderer;
+import com.vaadin.flow.data.renderer.LocalDateTimeRenderer;
+import com.vaadin.flow.data.value.ValueChangeMode;
+import java.io.Serial;
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+import life.qbic.datamanager.views.AppRoutes.Projects;
+import life.qbic.datamanager.views.general.PageArea;
+import life.qbic.projectmanagement.application.ProjectInformationService;
+import life.qbic.projectmanagement.application.ProjectPreview;
+import life.qbic.projectmanagement.application.SortOrder;
+
+/**
+ * <b>Project Collection</b>
+ * <p>
+ * A component that displays previews of accessible project previews.
+ * <p>
+ * The component also fires {@link ProjectCreationClickedEvent} to all registered listeners, if a
+ * user has the intend to add a new project.
+ *
+ * @since 1.0.0
+ */
+public class ProjectCollection extends PageArea {
+
+  private Div controlSection = new Div();
+  private Div gridSection = new Div();
+  private String projectPreviewFilter = "";
+  private GridLazyDataView<ProjectPreview> projectPreviewGridLazyDataView;
+  private Div titleSection = new Div();
+  @Serial
+  private static final long serialVersionUID = 8579375312838977742L;
+  final TextField projectSearchField = new TextField();
+  final Grid<ProjectPreview> projectGrid = new Grid<>(ProjectPreview.class, false);
+  final Button createProjectButton = new Button("Add");
+  private final String title;
+  private final ZoneId clientZoneId;
+  private final ProjectInformationService projectInformationService;
+  private final List<ComponentEventListener<ProjectCreationClickedEvent>> projectCreationClickedListeners = new ArrayList<>();
+
+  private ProjectCollection(String title, ZoneId clientZoneId,
+      ProjectInformationService projectInformationService) {
+    this.title = title;
+    this.clientZoneId = clientZoneId;
+    this.projectInformationService = projectInformationService;
+    layoutComponent();
+    createLazyProjectView();
+    configureSearch();
+    configureProjectCreationButton();
+  }
+
+  private void layoutComponent() {
+    addClassNames("projects-collection");
+    layoutSearchField();
+    layoutProjectCreationButton();
+    layoutGrid();
+    layoutTitleSection();
+    layoutControlSection();
+    layoutGridSection();
+    controlSection.add(new Div(projectSearchField));
+    controlSection.add(new Div(createProjectButton));
```

Comment:
I think this was a miscommunication on my part! 
You only need the controlsection as a the div container to which you can add the vaadin components themselves 
```suggestion
    controlSection.add(projectSearchField);
    controlSection.add(createProjectButton);
```